SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/DefaultApplicationSettings.cs

```diff
@@ -21,18 +22,33 @@ public DefaultApplicationSettings()
         {
             lock (configLock)
             {
-                configuration = OpenConfiguration();
+                try
+                {
+                    configuration = OpenConfiguration();
+                }
+                catch (Exception e)
```

Comment:
Either specify what exceptions are caught here, or adjust error message, since various exceptions might be thrown (Corrupt file, can't access, file not exists?..)

Reply:
Fixed